src/main/java/pl/itacademy/schedule/generator/ExcelCreator.java

```diff
@@ -0,0 +1,103 @@
+package pl.itacademy.schedule.generator;
+
+import org.apache.poi.hssf.usermodel.HSSFWorkbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.apache.poi.ss.usermodel.*;
+
+import static pl.itacademy.schedule.ScheduleGeneratorApp.DATE_FORMATTER;
+import static pl.itacademy.schedule.ScheduleGeneratorApp.TIME_FORMATTER;
+
+public class ExcelCreator {
+
+	private Workbook workbook;
+	private CellStyle cellStyleRight;
+	private CellStyle cellStyleLeft;
+	private CellStyle cellStyleCenter;
+
+	public Workbook createHSSFWorkbook(Schedule schedule) {

```

Comment:
Let's rid of this older format, and keep to newer one - XSSF

Reply:
OK